src/framework/preferences/css/stylus/utils/Themes.styl

```diff
@@ -241,6 +241,7 @@ build-themes-Enactors(contrastThemes) {
                     border-right-color: bColor !important;
                     border-bottom-color: bColor !important;
                     border-left-color: bColor !important;
+                    box-shadown: none !important; // Shadow Bug Fix
```

Comment:
Yes sure. I will fix it in few hours once I get home. I hope this wont
negatively effect on my proposal, which I will be sending tonight.

Regards,
Blagoj
On 18 Mar 2016 2:40 pm, "Justin Obara" <notifications@github.com> wrote:

> In src/framework/preferences/css/stylus/utils/Themes.styl
> <https://github.com/fluid-project/infusion/pull/691#discussion_r56656492>:
>
> > @@ -241,6 +241,7 @@ build-themes-Enactors(contrastThemes) {
> >                      border-right-color: bColor !important;
> >                      border-bottom-color: bColor !important;
> >                      border-left-color: bColor !important;
> > +                    box-shadown: none !important; // Shadow Bug Fix
>
> Could you please add the JIRA number to this comment as well.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/fluid-project/infusion/pull/691/files/acab904d495fbdbdae649cdf36b80509c1433b66#r56656492>
>